components/channel_layout/app_route/app_route.jsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import $ from 'jquery';
```

Comment:
We avoid to use jquery if is not needed (we are removing it from the code), and in this case we can easily replace the jquery code here with vanilla js

Reply:
done